web/src/rest/publish.js

```diff
@@ -40,9 +44,18 @@ const publishRest = new Vue({
     };
   },
   methods: {
-    // TODO proper OAuth login
+    async restoreLogin() {
+      await oauthClient.maybeRestoreLogin();
+      if (oauthClient.isLoggedIn) {
+        this.token = oauthClient.token;
```

Comment:
`.token` is not a public property and shouldn't be accessed anywhere by downstream code. Is there a particular reason you want to access it?

Reply:
A misguided attempt to keep the review footprint down.